src/components/content/order/scale/Scale.tsx

```diff
@@ -122,6 +126,40 @@ export const Scale = ({
         setScaleWarning('');
     };
 
+    const getFlavorTitle = (title: string): React.JSX.Element => {
+        return <span className={'flavor-card-title'}>{title}</span>;
+    };
+
+    const getFlavorContent = (flavor: ServiceFlavor): React.JSX.Element => {
```

Comment:
everything here seems to be duplicate of what we have in FlavorInfo.tsx. Please remove any duplicate code.